src/component/admin/billing/BillingGrid/BillingGrid.tsx

```diff
@@ -0,0 +1,325 @@
+import { FC, useMemo } from 'react';
+import { SxProps } from '@mui/system';
+import { Alert, Divider, Grid, styled, Theme, Typography } from '@mui/material';
+import { Link } from 'react-router-dom';
+import CheckIcon from '@mui/icons-material/Check';
+import { colors } from 'themes/colors';
+import useUsers from 'hooks/api/getters/useUsers/useUsers';
+import useInvoices from 'hooks/api/getters/useInvoices/useInvoices';
+import { BillingInfoButton } from './BillingInfoButton/BillingInfoButton';
+import { BillingHistory } from './BillingHistory/BillingHistory';
+import { useInstanceStatus } from 'hooks/api/getters/useInstanceStatus/useInstanceStatus';
+import { ConditionallyRender } from 'component/common/ConditionallyRender/ConditionallyRender';
+import { InstanceState } from 'interfaces/instance';
+
+const GridRow: FC<{ sx?: SxProps<Theme> }> = ({ sx, children }) => {
```

Comment:
Are these relevant for other components in the future? If so, maybe common is a better place?